benchmarks/ekobox/benchmark_evol_pdf.py

```diff
@@ -39,26 +40,27 @@ def benchmark_evolve_single_member(tmp_path, cd, lhapdf_path):
             "ktThr": 1,
         },
     )
+    nt, no = compatibility.update(theory, op)
     # lhapdf import (maybe i have to dump with a x*), do plots)
     with lhapdf_path(test_pdf):
         pdf = lhapdf.mkPDF("myCT14llo_NF3", 0)
     with cd(tmp_path):
         ev_p.evolve_pdfs(
             [pdf],
-            theory,
-            op,
+            nt,
+            no,
```

Comment:
I know they mean `new_theory_card` and `new_operator_card`, but it is a bit cryptic.
Please, favor more telling names. Trade-offs are best:

- ❌ `nt` - too short, ambiguous
- ❌ `new_theory_card_evolve_single_member` - redundant, it is already in the scope of `evolve_single_member`
- ✔️ `new_theory_card` - just perfect :)

The minimal amount of information that saves you an explicit comment is the optimal choice, according to me.

Reply:
I actually copied the name that was already there. Should I change also the others `nt`? (and similar for `no`)

Reply:
Not top priority, but I believe we should constantly improve also these details, they make it simpler for new developers and ourselves to maintain.

If you can improve something is already better than nothing, as I said it is not high-priority, don't waste too much time. But at least change this one, such that the new code has better and better style :)